### ISSUE TITLE
Chunk forceloading not working with Refined Storage

### ISSUE THREAD

**Issue by markaabo** [thexaero/open-parties-and-claims#329]
As the title says chunk force loading doesnt seem to be working with the mod Refined Storage.

I have a server running forge 1.20.1 with some mods and all mods are up to date as of today

The problem is when i leave my base and goto a very nearby base my refined storage monitors doesnt work and are turned off, even tho the controller and all power generation is in the main base and has chunks forceloaded by open claims, if another player is in the main base the monitors come on so it seems directly related to them not being loaded when no players are loading the base

if i can provide any more info just ask :)

**Comment by thexaero** [comment]
It might require players being nearby, not just chunks being loaded. Have you tried other mods that forceload chunks?
Have you made sure that the chunks are loaded for vanilla functions?

**Comment by markaabo** [comment]
> It might require players being nearby, not just chunks being loaded. Have you tried other mods that forceload chunks? Have you made sure that the chunks are loaded for vanilla functions?

I have not tried other chunk loader mods, but i have checked that vanilla stuff gets loaded, our automatic cactus farm and mobs seems to be loaded its just the refined storage or maybe mekanism which powers it.

**Comment by thexaero** [comment]
Please try a different claiming mod that has a forceloading function, e.g. FTB Chunks. If that also doesn't work, then it must really be directly checking for players.

**Comment by markaabo** [comment]
Okay so i tried using Mekanisms Anchor upgrade which chunk loads the chunk the machine its inserted in is in, and that works and makes the refined storage controller loaded, so with another chunk loader it works some how

**Comment by thexaero** [comment]
That would obviously work. Try a separate mod like FTB Chunks.

**Comment by markaabo** [comment]
> That would obviously work. Try a separate mod like FTB Chunks.

Well mekanism is not the same mod as refined storage so its litterally a seperate mod but anyway i have tried FTB Chunks and that works

**Comment by thexaero** [comment]
One of the options we had is that it could be Mekanism. Doesn't seem to be the case though.

Are you using network transmitters and receivers?
I tested some stuff myself and the problem with forceloading that I found is almost the same with Mekanism's anchor upgrade for me. It all works fine if you forceload both the transmitter and the receiver, which is what you're supposed to do. The problem stems from the receiver not being loaded when the transmitter looks for it. Not sure why it sometimes succeeds but it's not guaranteed with the anchor upgrade just on the transmitter end either. Seems like reconnecting power to the network causes a recheck, in case just forceloading the receiver doesn't work for you.
